packages/Interface/src/OperatorVector/DTK_IntegrationPointSet.hpp

```diff
@@ -80,6 +80,9 @@ class IntegrationPointSetIterator : public EntityIterator
     // Pre-increment operator.
     EntityIterator &operator++() override;
 
+    // Post-increment operator.
+    using EntityIterator::operator++;
```

Comment:
It does not by default. If you have overloaded operator, the override version in inherited class hides all others by default. [1](javascript override overload function) [2](http://stackoverflow.com/questions/1628768/why-does-an-overridden-function-in-the-derived-class-hide-other-overloads-of-the)